File: POO/Personal/TFT Champs/Java/Champ.java
class Champ {
    String name;
    Integer cost;
    Traits type;
        public Champ(String name, Integer cost, Traits type){
        this.name = name;
        this.cost = cost;
        this.type = type;
    }
    void printDataChamp(){
        System.out.println("El campeón es: " + name);
        System.out.println("Su primera clase: " + type.primeraClase);
        System.out.println("Su segunda clase: " + type.segundaClase);
    }
}

File: POO/Personal/TFT Champs/JavaScript/champ.js
class Champ {
    constructor(name, cost, traits) {
        this.name = name;
        this.cost = cost;
        this.traits = traits;
    }
    printDataChamp(){
        console.log("El campeón tiene como nombre: "+this.name);
        console.log(this.traits); //Object object, pero no dice mas
        console.log("El primero trait es: "+this.traits.primeraClase);
        console.log("El segunda trait es: "+this.traits.segundaClase);
    }
}